packages/components/src/radio-group/types.ts

```diff
@@ -0,0 +1,13 @@
+export type RadioGroupProps = {
+	label: string;
+	checked?: string | number;
+	defaultChecked?: string | number;
+	disabled?: boolean;
+	onChange?: ( value: string | number | undefined ) => void;
+	children: React.ReactNode;
+};
+
+export type RadioProps = {
+	value: string | number;
```

Comment:
Looking at the README, it seems that only string values were accepted. Should keep it like that?

In that case, we should also update `onChange`.

Reply:
That's interesting because there's a story which uses numbers. I'm all for only allowing strings though 👍🏻 I'll adjust the code accordingly.